backend/src/main/java/de/jmpsoftware/backend/controller/FaderPageController.java

```diff
@@ -41,17 +40,23 @@ public void getSingleValue(@RequestBody FaderItem faderItem, Principal principal
     @GetMapping(path = "/simplepage")
     public List<FaderItem> getSimplePage(Principal principal) {
         List<FaderItem> tmpList = new ArrayList<>();
-        for (int i = 0; i <= 10; i++ ) {
+        for (int i = 0; i <= 5; i++ ) {
```

Comment:
Why cahnge from 10 to 5? Try not to use "MagicNumbers" maybe save the value to a `int pageCount `or a constant
I would move the whole function in a service class

Reply:
Das ist nur eine Demo Liste damit ich damit arbeiten kann die ich immer anpassen kann.